### PR TITLE
Replace deprecated mime_content_type with finfo. Fixes #228

### PR DESCRIPTION
**Pull request by aleemb**

### CODE REVIEW

**Comment by miripiruni** [comment]
I see it's not working in PHP < 5.3.0...

**Comment by aleemb** [comment]
@miripiruni I have added backward compatibility for older versions.

File: src/cli.php [hunk 1]
```diff
@@ -86,6 +86,18 @@ function tool($argc, $argv) {
             }
             $c = new csscomb('', false, $sort);
 
+            $mime_type = '';
+            if (is_file($this->in)) {
+                if (function_exists('finfo_open')) {
+                    $finfo = finfo_open(FILEINFO_MIME_TYPE);
+                    $mime_type = finfo_file($finfo, $this->in);
```
Comment (miripiruni) [inline src/cli.php:93]: finfo_close ?

Comment (aleemb) [inline src/cli.php:93]: Sorry about that, starting to feel like a real slacker now. Thanks for catching that.

**Comment by tonyganch** [comment]
@miripiruni, are we fine with this PR? Can it be merged?

**Comment by miripiruni** [comment]
Okay.

**Comment by tonyganch** [comment]
@aleemb, thanks a lot!

**Pull request merged**